tests/Unit/DataStructures/Tensor/Expressions/Test_SquareRoot.cpp

```diff
@@ -0,0 +1,92 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <cstddef>
+#include <iterator>
+#include <numeric>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Expressions/Evaluate.hpp"
+#include "DataStructures/Tensor/Expressions/Product.hpp"
+#include "DataStructures/Tensor/Expressions/SquareRoot.hpp"
+#include "DataStructures/Tensor/Expressions/TensorExpression.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Utilities/MakeWithValue.hpp"
+
+namespace {
+template <typename... Ts>
+void assign_unique_values_to_tensor(
```

Comment:
Would it be useful to factor this  initialization code (in a separate PR) into a helper file? E.g. into `tests/Helpers/DataStructures/Tensor/Expressions/AssignUniqueValues.hpp`?

Reply:
Yah, this definitely needs to be done, and I even started doing this locally! I can submit a PR for it after this PR is in.  Do you think it would make more sense to put in `tests/Helpers/DataStructures/Tensor/` since it is useful for testing `Tensor`s overall, or would you prefer it to be in `tests/Helpers/DataStructures/Tensor/Expressions/`?

Reply:
:+1: `tests/Helpers/DataStructures/Tensor/` sounds like the right place to me :)